### ISSUE TITLE
Error on export: Multi-valued field cannot be used in ORDER BY clause

### ISSUE THREAD

**Issue by Falthazar** [timabell/msaccess-vcs-integration#61]
It looks like it isn't exporting the multi-valued field data correctly.

I'm getting the error in `VCS_ExportTableData` on this line:
`Set rs = CurrentDb.OpenRecordset(TableExportSql(tbl_name))`

Stepping through, it looks like it is building the SQL query, but you can't order by a multi-value field. 

The issue is specifically a value list - as in each option is just typed in row source.

Anyone know how to fix this?

**Comment by hangie** [comment]
I also had this issue and have implemented a change to support this.  Essentially using the XML import/export approach.  Changes are on https://github.com/hangie/msaccess-vcs-integration/tree/feature/import_data_update

Have a look and if you like it, then I can go through the pull requests to get it integrated.

**Comment by timabell** [comment]
I'll assume the merged change fixed this, let us know if it didn't!

**Comment by hangie** [comment]
No, this one isn't closed until I send it back.  The merge was on my repository - sorry.

I'll add a pull request for this one shortly.
